inlang/source-code/badge/src/main.ts

```diff
@@ -44,6 +44,6 @@ if (isProduction) {
 app.set("base", "/")
 app.use(router)
 
-const port = process.env.PORT ?? 4002
```

Comment:
in case this is a misunderstanding i meant hard coding render to use the port that is configured for the services not removing the PORT env variable from the services.

Reply:
My thought was, if every server has this setting, the fastest server will serve on this port and we will have the same situation as before.

Reply:
@NiklasBuchfink  ahh good point. only the server that will be exposed can have this setting